server/controllers/index.js

```diff
@@ -182,12 +182,20 @@ export const explore = async(ctx, next) => {
   };
 
   const aDropInTheOceanPromoList = PromoListFactory.fromSeries(aDropInTheOceanSeries);
+  const digitalStories = commissionedSeries.filter(s => s.commissionedLength);
```

Comment:
Run me through what you're doing here in person perhaps?
This is a strange part of the controller anyway, so potentially fine.